### PR TITLE
Pooltogether Polygon connector

### PR DESCRIPTION
**Pull request by eccheung4**
Updated contracts for polygon, but pretty much the same.
- Removed code for pods as there are no pods on polygon.
- Updated to use convertWmaticToMatic
Updated tests to use AAVE and USDC Sponsor pool on polygon

Pool Together Connector that will allow DSA to:

depositTo : Deposit into a Prize Pool
withdrawInstantlyFrom: Withdraw from a Prize Pool
claim: Claim from a token faucet
claimAll: Claim from multiple token faucet

### CODE REVIEW

File: contracts/polygon/connectors/pooltogether/main.sol [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        TokenFaucetProxyFactoryInterface tokenFaucetProxyFactoryContract = TokenFaucetProxyFactoryInterface(tokenFaucetProxyFactory);
+
+        tokenFaucetProxyFactoryContract.claimAll(address(this), tokenFaucets);
+
+        _eventName = "LogClaimAll(address,address,TokenFaucetInterface[])";
+        _eventParam = abi.encode(address(tokenFaucetProxyFactory), address(this), tokenFaucets);
+    }
+}
+
+contract ConnectV2PoolTogether is PoolTogetherResolver {
```
Comment (thrilok209) [inline contracts/polygon/connectors/pooltogether/main.sol]:
```suggestion
contract ConnectV2PoolTogetherPolygon is PoolTogetherResolver {
```

**Comment by thrilok209** [comment]
@eccheung4 Connector looks good! Could you deploy the connector and post it over here?

**Comment by eccheung4** [comment]
@thrilok209 Deployed Connector

ConnectorName: "POOLTOGETHER-A"
ConnectorAddress: [0xbf254fA556703c78d70B4Cec8459d1Ff2b33Cd87](https://polygonscan.com/address/0xbf254fA556703c78d70B4Cec8459d1Ff2b33Cd87#code)

**Comment by thrilok209** [comment]
Connector whitelisted: https://polygonscan.com/tx/0xd2ff8093c397661270a02556595e47b9567c93d5ca8369757c9b02cfafad6874

**Review by thrilok209 — APPROVED** [review]

**Pull request merged**